cron-jobs/functions/sendNewAnswer.js

```diff
@@ -1,18 +1,23 @@
-import {Config} from "../../config/config.js";
-const config = new Config()
+import { Config } from "../../config/config.js";
+const config = new Config();
 export async function sendNewAnswer(channel) {
-    const apiUrl = config.questionsApiUrl()
-    const answer = await fetch(apiUrl)
-    const answerBody = await answer.json()
-    switch (answerBody.correctAnswer) {
-        case 'A':
-            channel.send(`<:training_team:895480894901592074> The correct answer is ||🇦 ${answerBody.optionA}||`)
-            break
-        case 'B':
-            channel.send(`<:training_team:895480894901592074> The correct answer is ||🇧 ${answerBody.optionB}||`)
-            break
-        case 'C':
-            channel.send(`<:training_team:895480894901592074> The correct answer is ||🇨 ${answerBody.optionC}||`)
-            break
-    }
-}
\ No newline at end of file
+  const answer = await fetch(`${config.quizBaseUrl()}/api/quiz/current`)
+  const answerBody = await answer.json();
+  switch (answerBody.correctAnswer) {
+    case "A":
+      channel.send(
+        `<:training_team:895480894901592074> The correct answer is ||🇦 ${answerBody.optionA}||`,
+      );
+      break;
+    case "B":
+      channel.send(
+        `<:training_team:895480894901592074> The correct answer is ||🇧 ${answerBody.optionB}||`,
+      );
+      break;
+    case "C":
+      channel.send(
+        `<:training_team:895480894901592074> The correct answer is ||🇨 ${answerBody.optionC}||`,
+      );
+      break;
+  }
```

Comment:
Get rid of the switch. 

```
const options = {
"A": answerBody.optionA,
"B":answerBody.optionB,
"C":answerBody.optionC

channel.send(
        `<:training_team:895480894901592074> The correct answer is ||🇨 ${options[answerBody.correctAnswer]||`,
      );
}
```

Reply:
Without the switch, how are we supposed to change the emoji? We could make an answerEmojis object, but I feel like at that point it's better to just have the switch. Is there any harm in having the switch/does it make it slower?